improc/alignment.py

```diff
@@ -350,7 +366,7 @@ def _align_swarp( self, image, target, sources, target_sources ):
                         '-WRITE_XML', 'N' ]
 
             t0 = time.perf_counter()
-            res = subprocess.run( command, capture_output=True, timeout=60 )
+            res = subprocess.run(command, capture_output=True, timeout=self.pars.scamp_timeout)
```

Comment:
This should be `self.pars.swarp_timeout`

Above, line 305, in the call `swarp_fodder_wcs = improc.scamp.solve_wcs_scamp(`, you should add passing self.pars.scamp_timeout as an argument.